src/shared/helpers/user-role.ts

```diff
@@ -0,0 +1,17 @@
+import { Avo } from '@viaa/avo2-types';
+
+import { ShareRightsType } from '../components/ShareWithColleagues/ShareWithColleagues.types';
+import { Contributor } from '../types/contributor';
+
+export const getUserRoleType = (
```

Comment:
To avoid confusing this with roles from ldap and usergroups
i would prefer this naming:
```suggestion
export const getContributorType = (
```